script.js

```diff
@@ -0,0 +1,27 @@
+//Utworzenie funkcji konstruującej Button
+
+function Button(text) {
+	this.text = text || 'Hello';
+}
+
+//Utworzenie podstawowej metody create tworzącej przycisk na stronie
+
+Button.prototype = {
+	create: function() {
+		const self = this;					//wskazuje kontekst, zapobiega utracie kontekstu
```

Comment:
why `const` not `var`?

Reply:
Is it wrong? This variable won't change, therefore I thought it is good to use it here.